src/ChirpInfrastructure/Migrations/ChirpDBContextModelSnapshot.cs

```diff
@@ -19,32 +19,73 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("ChirpCore.Domain.Author", b =>
                 {
-                    b.Property<int>("AuthorId")
+                    b.Property<int>("Id")
                         .ValueGeneratedOnAdd()
                         .HasColumnType("INTEGER");
 
+                    b.Property<int>("AccessFailedCount")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("ConcurrencyStamp")
+                        .IsConcurrencyToken()
+                        .HasColumnType("TEXT");
+
                     b.Property<string>("Email")
+                        .HasMaxLength(256)
                         .HasColumnType("TEXT");
 
-                    b.Property<string>("Name")
-                        .IsRequired()
+                    b.Property<bool>("EmailConfirmed")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<bool>("LockoutEnabled")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<DateTimeOffset?>("LockoutEnd")
                         .HasColumnType("TEXT");
 
-                    b.Property<string>("Nickname")
+                    b.Property<string>("NormalizedEmail")
+                        .HasMaxLength(256)
                         .HasColumnType("TEXT");
 
-                    b.HasKey("AuthorId");
+                    b.Property<string>("NormalizedUserName")
+                        .HasMaxLength(256)
+                        .HasColumnType("TEXT");
 
-                    b.HasIndex("AuthorId")
```

Comment:
why is this AuthorId becoming PasswordHash, compared to above NormalizedUserName, 
I get it is probably correct and Migrations makes this, but I am wondering :)

Reply:
Good question, I think it's just the auto generated code from migrations that looks different now. It shouldn't mean anything, but I could be wrong.